Fails to start with error flag "client-secret-file" does not exist

After upgrading the docker image this morning my otherwise working instance fails to start with the error: flag "client-secret-file" does not exist

Seems this is related to #341 and #355 

```
2020/02/15 16:00:33 ERROR: flag "client-secret-file" does not exist
panic: ERROR: flag "client-secret-file" does not exist

goroutine 1 [running]:
log.Panicf(0xb6d3da, 0x1d, 0xc000233cc8, 0x1, 0x1)
/usr/local/go/src/log/log.go:345 +0xc0
github.com/mreiferson/go-options.Resolve(0xa74860, 0xc0000d7600, 0xc0000a5200, 0xc0000f74d0)
/go/pkg/mod/github.com/mreiferson/go-options@v0.0.0-20190302064952-20ba7d382d05/options.go:73 +0xb7b
main.main()
/go/src/github.com/pusher/oauth2_proxy/main.go:159 +0x1e13
```

## Expected Behavior

The lack of this config option shouldn't cause a complete failure to start.

## Steps to Reproduce (for bugs)

1. Don't configure  the 'client-secret-file' flag.
2. Start oauth2
3. Fails to start

## Your Environment

Docker image on unRaid OS.

- Version used: [latest docker image](https://quay.io/repository/pusher/oauth2_proxy/manifest/sha256:78f2205712887c5eff98e515c3a99f8f84db4ca6458042e1d4a9b32844427d6d)

--- a/main.go
+++ b/main.go
@@ -70,6 +70,7 @@ func main() {
 	flagSet.String("google-service-account-json", "", "the path to the service account json credentials")
 	flagSet.String("client-id", "", "the OAuth Client ID: ie: \"123456.apps.googleusercontent.com\"")
 	flagSet.String("client-secret", "", "the OAuth Client Secret")
+	flagSet.String("client-secret-file", "", "the file with OAuth Client Secret")
 	flagSet.String("authenticated-emails-file", "", "authenticate against emails via file (one per line)")
 	flagSet.String("htpasswd-file", "", "additionally authenticate against a htpasswd file. Entries must be created with \"htpasswd -s\" for SHA encryption or \"htpasswd -B\" for bcrypt encryption")
 	flagSet.Bool("display-htpasswd-form", true, "display username / password login form if an htpasswd file is provided")

